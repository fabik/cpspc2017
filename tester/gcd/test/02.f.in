15
28 23 26 12 15 9 9 30 23 16 10 10 9 22 4
