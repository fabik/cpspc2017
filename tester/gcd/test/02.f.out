1
31
59
85
108
131
153
169
184
196
206
218
228
237
246
